apps/cowswap-frontend/src/modules/tokensList/pure/ChainsSelector/index.tsx

```diff
@@ -38,6 +50,18 @@ export function ChainsSelector({
   const isMobile = useMediaQuery(Media.upToSmall(false))
 
   const theme = useTheme()
+  const tradeTypeInfo = useTradeTypeInfoFromUrl()
+  const mode =
```

Comment:
Why mapping this?
Can `tradeTypeInfo?.tradeType || 'unknown'` be used instead?

Reply:
> Addressed: ChainsSelector now receives tradeType via props, no hook call and we map known enum values (SWAP, LIMIT_ORDER, ADVANCED_ORDERS) to the string modes the analytics label expects. unknown remains the fallback.